doc/taskwiki.txt

```diff
@@ -382,8 +382,22 @@ viewport definition:
 
 ~   == Differently ordered home tasks | project:Home $T ==
 
+------------------
+5.2.4 Limiting number of tasks displayed
+
+By default, taskwiki will present the full list of tasks that match the
+viewport definition.
+
+You can limit the number of tasks listed by using the limit filter after the
+sorting identifier:
+
+~   == List only the 5 first tasks | project:Home limit:5 ==
```

Comment:
This example doesn't really demonstrate that `limit:5` must come after `#X` or `$Y`, and makes it look like `limit:5` is just another taskwarrior filter (which is what I think @tbabej meant by “just "overriding" the behavior of the limit:<limit> clause that TW users are already familiar with”). I'm afraid this will confuse people.

Reply:
I clarified the docs. There is the issue that it looks a bit unnatural to have sth like:

```
# Tasks | project:mine other:filter +tag1 $T limit:3
```

I would expect taskwiki stuff to be last and the above seems like taskwarrior syntax is split.